### PR TITLE
Feature/gm redesign part 1

### PR DESCRIPTION
**Pull request by kirilknysh**
image
image

### CODE REVIEW

File: public/common/components/players-list/styles/players-list.styl [hunk 1]
```diff
@@ -1,38 +1,57 @@
 @import '../../../stylesheets/color-palette'
+@import '../../../stylesheets/common'
```
Comment (DaQuirm) [inline public/common/components/players-list/styles/players-list.styl]: We should create a webpack alias for stylesheets...

Comment (kirilknysh) [inline public/common/components/players-list/styles/players-list.styl]: @DaQuirm how about to start with `@import '~common/stylesheets/common'` ?

File: public/common/components/players-list/view-models/players-list.viewmodel.coffee [hunk 2]
```diff
@@ -1,4 +1,21 @@
 class UserPanelViewModel
```
Comment (DaQuirm) [inline public/common/components/players-list/view-models/players-list.viewmodel.coffee]: Rename the class pls 🐱

File: public/common/components/players-list/view-models/players-list.viewmodel.coffee [hunk 3]
```diff
@@ -1,4 +1,21 @@
...
 	constructor: (@players) ->
+		@totalCount = new nx.Cell
+			'<-': [
+				@players,
+				(players) -> if players? then players.length else 0
```
Comment (kirilknysh) [inline public/common/components/players-list/view-models/players-list.viewmodel.coffee]: Please check is `@players` is always collection and then use `@players.length`

**Pull request merged**